tests/resources/conftest.py

```diff
@@ -157,15 +157,15 @@ def resource_type_v(app, resource_type_type):
 
 
 RunningApp = namedtuple("RunningApp", [
-    "app", "location", "resource_type_v"
+    "app", "location", "resource_type_v", "laguanges_v", "subject_v"
```

Comment:
It was needed cuz otherwise any search would fail to find them in DB and report back a 404... This also means we are somehow doing DB queries for vocabularies (languages at least) when searching @lnielsen 

- To be fixed in another issue...

Reply:
Do you have a trace of them?

Reply:
Sadly not... but if you remove the two vocabularies from this fixture it fails with 404. Going in depth you will see a `NoResultFound` from the DB...

I can debug this before starting with the frontend, since it might affect us severly....

Reply:
@lnielsen Thanks to @jrcastro2 
```
Traceback (most recent call last):
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/flask_resources/resources.py", line 65, in view
    return view_meth()
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/flask_resources/content_negotiation.py", line 116, in inner_content_negotiation
    return f(*args, **kwargs)
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/flask_resources/parsers/decorators.py", line 51, in inner
    return f(self, *args, **kwargs)
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/flask_resources/responses.py", line 39, in inner
    res = f(*args, **kwargs)
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/invenio_records_resources/resources/records/resource.py", line 80, in search
    return hits.to_dict(), 200
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/invenio_records_resources/services/records/results.py", line 196, in to_dict
    if self.aggregations:
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/invenio_records_resources/services/records/results.py", line 153, in aggregations
    return self._results.labelled_facets.to_dict()
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/invenio_records_resources/services/records/facets/response.py", line 82, in labelled_facets
    data, selection)
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/invenio_records_resources/services/records/facets/facets.py", line 56, in get_labelled_values
    label_map = self.get_label_mapping(data.buckets)
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/invenio_records_resources/services/records/facets/facets.py", line 34, in get_label_mapping
    return value_labels(keys)
  File "/home/javier/cern/invenio/modules/invenio-vocabularies/invenio_vocabularies/services/facets/labels.py", line 44, in __call__
    identity, type=self.vocabulary, ids=ids, fields=self.fields)
  File "/home/javier/cern/invenio/modules/invenio-vocabularies/invenio_vocabularies/services/service.py", line 177, in read_many
    vocabulary_type = VocabularyType.query.filter_by(id=type).one()
  File "/home/javier/.pyenv/versions/3.6.7/envs/invenioRDM/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3500, in one
    raise orm_exc.NoResultFound("No row was found for one()")
sqlalchemy.orm.exc.NoResultFound: No row was found for one()
127.0.0.1 - - [23/Jun/2021 11:05:47] "GET /api/records?q=&sort=newest&page=1&size=10 HTTP/1.1" 404
```